### DISCUSSION TITLE
Fluent-Bit in My Personal Project

### DISCUSSION THREAD

**Discussion by garutialle** [fluent/fluent-bit#8769] [Q&A]
Hi, I'm implementing fluent-bit in one of my projects. I monitor the status of some operations and write a log file where for each line I print the execution status of my processes

proc1 111111
proc2 222222
proc3 333333
proc1 111111
proc2 222222
proc3 333333

With fluent-bit I created an input tail that keeps track of this file and it's great because I perfectly translate the status of the message, identify the part of the message and translate it to active or inactive. But now I have to store and check the status somewhere. In the sense that I have to intercept the change of state of a process. If it goes from active to inactive. I found a checklist plugin but I can't handle entering and exiting an item from the list

Thank you

**Comment by patrick-stephens** [comment]
It sounds more like you're making metrics so it might be better to follow that approach - there's even a logs to metric plugin if you want. Then the downstream thing monitoring your metrics can handle those transitions, e.g. Prometheus or AlertManager.

Otherwise what do you want Fluent Bit to do with those values? Normally the observability stack downstream handles this, Fluent Bit is just the means to expose the values appropriately in whatever format that wants.
